doc/source/scipyoptdoc.py

```diff
@@ -60,7 +60,7 @@ class ScipyOptimizeInterfaceDomain(PythonDomain):
     def __init__(self, *a, **kw):
         super().__init__(*a, **kw)
         self.directives = dict(self.directives)
-        self.directives['function'] = wrap_mangling_directive(self.directives['function'])
+        self.directives['function'] = wrap_mangling_directive(self.directives['function'])  # noqa: E501
```

Comment:
any nice way to split this or no?

Reply:
I'd either store `wrap_mangling_directive(self.directives['function'])` or just `self.directives['function']` in a variable with a short enough name, then use the variable on this line. Could also break the line after `wrap_mangling_directive(` and single indent the next line.